lib/config.js

```diff
@@ -169,6 +169,9 @@ config.workspaceHostUnhealthyTimeoutSec = 12 * 60 * 60;
 config.workspaceHostLaunchTimeoutSec = 10 * 60;
 config.workspaceUrlRewriteCacheMaxAgeSec = 60 * 60;
 config.workspacePercentMessageRateLimitSec = 1;
+config.workspaceHomeDirRoot = '/jobs/workspaces'; /* Where the main server will store workspace files.  Note on production this should be different from the jobs directory, this is just to make local development easier. */
+config.workspaceHostHomeDirRoot = '/jobs/workspaces'; /* Where the workspace host server will store workspace files. */
+config.workspaceHostWatchJobFiles = false; /* Should be enabled if we're watching files to sync with S3 */
```

Comment:
For backwards compatibility should we set this to `true` by default?

If I'm understanding correctly, when we first deploy this we'll be running with some workspaces backed by S3 and some by FileSystem (on the same workspace host). So we'll need to run the watcher, but if it detects a file change then we'll only take action if the file is for a workspace with S3 backing. After we are sure we no longer need workspaces with S3 backing (after the semester is over?) we'll set `workspaceHostWatchJobFiles = false`.

Reply:
Yes agreed!  We should set this to be `true` when we're running on production servers.  I have this set to `false` because I figured it's unnecessary when running locally on docker.